lambda/trmc.ml

```diff
@@ -0,0 +1,515 @@
+open Lambda
+
+open struct
```

Comment:
Oh, is this used to define functions local to the module which will not be exported even though there is no `.mli`?  If so that is a neat trick I haven't seen used yet.

Reply:
Yes, it's a recent idiom given that `open M` didn't accept `struct .. end` module expressions before 4.08.